backend/tests/conftest.py

```diff
@@ -135,10 +135,10 @@ def open(self, *args, **kwargs):
 @pytest.fixture
 def new_msg():
     msg = {
-        "name": "TestName",
-        "email": "test@test.com",
```

Comment:
test.com is a real domain, we don't want to send any mail there :) `example.com` is recommended afaik.